### PR TITLE
add action and status services

### PR DESCRIPTION
**Pull request by davidjamesherzog**
@jinpark - can't really test whether or not I have it correct or not

### CODE REVIEW

**Comment by jin** [comment]
Please use the correct @mention @davidjamesherzog :)

**Comment by jinpark** [comment]
sorry jin!

File: src/client/app/services/poker.service.js [hunk 1]
```diff
@@ -11,11 +11,34 @@
...
     var poker = {
+      action: action,
       getHand: getHand,
-      join: join
+      join: join,
+      status: status
     };
     return poker;
 
+    function action(action, amount) {
```
Comment (jinpark) [inline src/client/app/services/poker.service.js:21]: Is the action endpoint just going to be for bet related actions?

Comment (jinpark) [inline src/client/app/services/poker.service.js:21]: Just verified that its going to be `fold` and `call`

**Comment by jinpark** [comment]
LGTM @davidjamesherzog

**Pull request merged**